client/src/components/Menu.js

```diff
@@ -0,0 +1,30 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import Header from './Header';
+import AppContainer from './AppContainer';
+import { MenuButton, MenuButtonTransparent } from './MenuButtons';
+
+const MenuContainer = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
```suggestion
  align-items: center;
  & > button {
  background-color: rgb(219, 143, 0);
  }
  & :nth-child(3n) {
  background-color: rgb(219, 143, 0, 63%);
  }
```

this could also give your `MenuButton` components different backgrounds